app/Console/Commands/FetchTweetsCommand.php

```diff
@@ -65,39 +65,32 @@ public function __construct()
      * @return mixed
      */
     public function handle()
-    {        
-        $query = implode(' OR ', self::SEARCH_TERMS);
-        $query .= ' -filter:retweets'; // Filter out retweets
-
-        // q: 500 characters max
-        // count: max 100 results
-        // https://dev.twitter.com/rest/reference/get/search/tweets
-        
-        $data = Twitter::getSearch([
-            'q' => $query,
-            'result_type' => 'recent',
-            'count' => 100
-        ]);
+    {
 
-        $insertsCount = 0;
+        TwitterStreamingApi::publicStream()
+            ->whenHears(self::SEARCH_TERMS, function ($status) {
+                // Skip retweets
+                if (($status['retweeted_status'] ?? false) || mb_strpos($status['text'], 'RT ') === 0) {
```

Comment:
We might actually be able to keep retweets, now that we're using the streaming API.

I filtered them out before, because we were only able to get 100 statuses at the time, so I wanted to avoid any extra noise.

Reply:
@MiniCodeMonkey I thought about that, but I think it could overwhelm the server pretty easily (could be talking about thousands of RTs for even a single tweet). We could perhaps keep manual RTs, but I don't think auto-RTs could provide any additional information, right?